<?xml version="1.0"?>
<launch>
  <include file="$(find akagachi_demo)/launch/rcservo_minimal.launch"/>

  <node name="rviz" pkg="rviz" type="rviz" output="screen" args="-d '$(find akagachi_demo)/config/layout.rviz'"/>
  <node name="foveation_calculator" pkg="akagachi_demo" type="foveation_calculator.py" output="screen"/>
  <node name="vision_optical_flow" pkg="akagachi_demo" type="vision_optical_flow.py" output="screen"/>
  </launch>